front-end/src/screens/Settings/email.tsx

```diff
@@ -82,7 +103,7 @@ const Email = ({ className }: Props): JSX.Element => {
 					{error && <FilteredError text={error.message}/>}
 					{!currentUser.email_verified &&
 				<div className={'warning-text'}>
-					<div><Icon name='warning circle' />Your email is not verified.</div>
+					<div><Icon name='warning circle' />Your email is not verified. <a href='#' onClick={handleResendVerifyEmailTokenClick}>Resend verification email.</a></div>
```

Comment:
Let's show this warning and link only if the email isn't empty. We should also use the same style for links than what is used on the "forgor your password" link on the login form (except the font size, it's pretty big on the login form).

Reply:
done